code/index.html

```diff
@@ -3,28 +3,188 @@
+                  </p>
+                </div>
+              </div>
+          </div>
+
+          <div class="accordion-items">
+            <div class="accordion-question" id="section3">
+              What should I wear?
+            </div>
+            <div class="accordion-answer">
+              <div class="accordion-answer-content">
+                <p>
+                  On Mondays: Wear colored shoes On Tuesdays: Wear shirts with
+                  slogans On Wednesdays: Wear pink On Fridays: Wear jeans or
+                  track pants • Never wear strappy shoes with a mini-skirt.
+                  Don't wear hoop earrings.
+                </p>
+              </div>
+            </div>
+          </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- Signup form -->
+      <section class="form-box">
+        <form action="http://httpbin.org/anything" method="POST">
+          <h2>Sign up here!</h2>
+          <label for="name">
+            Name:
+            <input type="text" id="name" name="name" placeholder="Name" />
+          </label>
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              id="email"
+              name="email"
+              placeholder="example@gmail.com"
+              required
+            />
+          </label>
+          <label for="phoneNumber">
+            Phone Number
+            <input
+              type="tel"
+              pattern="[0][7][0-9]{1}[0-9]{4}[0-9]{3}"
+              maxlength="10"
+              required
+              id="phoneNumber"
+              name="phoneNumber"
+              placeholder="0700000000"
+            />
+          </label>
+          <label class="terms" for="terms">
+            <input
+              class="checkbox"
+              type="checkbox"
+              id="scales"
+              name="scales"
+              checked
+            />
+            Grool! I agree to the Terms & Condition
+          </label>
+          <button class="form-button" type="submit" value="sign-up">
+            Sign-up
+          </button>
+        </form>
+      </section>
+    </main>
+
+      <footer>
+        <h2>Laboratorie</h2>
+        <div class="social-media-icons">
+          <a href=""></a>
+          <a href=""></a>
+          <a href=""></a>
+          <a href=""></a>
+        </div>
+        <div class="address">
+          1 Kätleen, 111 11 Stockholm, Sweden 08-111-11
+        </div>
+      </footer>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="arrow-up-box">
+        <a href="" class="arrow-up">
+          &#8673
+        </a>
       </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
+      <script src="https://code.jquery.com/jquery-3.6.0.slim.min.js" integrity="sha256-u7e5khyithlIdTpu22PHhENmPcRdFiHRjhAuHcs05RI=" crossorigin="anonymous"></script>
```

Comment:
wow, so you got some other js-link as well? will definitely ask you in person what this one does!